source/SpinalHDL/Simulation/install.rst

```diff
@@ -55,6 +55,19 @@ Then you should be able to run SpinalSim + verilator from your Scala project wit
 
 From a fresh install of MSYS2 MinGW 64-bits, you will have to run the following commands inside the MSYS2 MinGW 64-bits shell (enter commands one by one):
 
```

Comment:
I think a subparagraph is necessary to distinguish the step.. 
```suggestion

from the minGW packet manager
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

```

Reply:
I can't agree with you more